internals/secrethub/app.go

```diff
@@ -74,6 +75,9 @@ type App struct {
 // newClientFunc creates a ClientAdapater.
 type newClientFunc func() (secrethub.ClientInterface, error)
 
+// newWriterFunc creates a WriterAdapter
+type newWriterFunc func(filename string, data []byte, perm os.FileMode) error
+
```

Comment:
I don't think it is necessary to declare a separate type for this function as it is only used in one place.

I'd say, for the sake of consistency we should drop this type.

Reply:
Fixed